testing/internal/e2e/boundary/scopes.go

```diff
@@ -25,21 +25,47 @@ func CreateNewOrgApi(t testing.TB, ctx context.Context, client *api.Client) stri
 	return newOrgResult.Item.Id
 }
 
+// CreateNewProjectApi creates a new project in boundary using the go api.
+// Returns the id of the new project.
+func CreateNewProjectApi(t testing.TB, ctx context.Context, client *api.Client, scopeId string) string {
```

Comment:
This should explain what the `scopeId` parameter is for. I think it's the ID of the org you want to create the project in?
```suggestion
// CreateNewProjectApi creates a new project in boundary using the go api.
// Returns the id of the new project.
func CreateNewProjectApi(t testing.TB, ctx context.Context, client *api.Client, orgScopeId string) string {
```